app/client/cli/actor.go

```diff
@@ -0,0 +1,318 @@
+				ServiceUrl: serviceURI,
+				Signer:     pk.Address(),
+				ActorType:  cmdDef.ActorType,
+			}
+
+			tx, err := prepareTxJson(msg, pk)
+			if err != nil {
+				return err
+			}
+
+			resp, err := postRawTx(cmd.Context(), pk, tx)
+			if err != nil {
+				return err
+			}
+			// DISCUSS(#310): define UX for return values - should we return the raw response or a parsed/human readable response? For now, I am simply printing to stdout
+			fmt.Println(resp)
+
+			return nil
+		},
+	}
+	return editStakeCmd
+}
+
+func newUnstakeCmd(cmdDef actorCmdDef) *cobra.Command {
+	unstakeCmd := &cobra.Command{
+		Use:   "Unstake <fromAddr>",
+		Short: "Unstake <fromAddr>",
+		Long:  fmt.Sprintf(`Unstakes the prevously staked tokens for the %s actor with address <fromAddr>`, cmdDef.Name),
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(#150): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+
+			// TODO (team): passphrase is currently not used since there's no keybase yet, the prompt is here to mimick the real world UX
+			pwd = readPassphrase(pwd)
+
+			msg := &typesUtil.MessageUnstake{
+				Address:   pk.Address(),
+				Signer:    pk.Address(),
+				ActorType: cmdDef.ActorType,
+			}
+
+			tx, err := prepareTxJson(msg, pk)
+			if err != nil {
+				return err
+			}
+
+			resp, err := postRawTx(cmd.Context(), pk, tx)
+			if err != nil {
+				return err
+			}
+			// DISCUSS(#310): define UX for return values - should we return the raw response or a parsed/human readable response? For now, I am simply printing to stdout
+			fmt.Println(resp)
+
+			return nil
+		},
+	}
+	return unstakeCmd
+}
+
+func newUnpauseCmd(cmdDef actorCmdDef) *cobra.Command {
+	unpauseCmd := &cobra.Command{
+		Use:   "Unpause <fromAddr>",
+		Short: "Unpause <fromAddr>",
+		Long:  fmt.Sprintf(`Unpauses the %s actor with address <fromAddr>`, cmdDef.Name),
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(#150): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+
+			// TODO (team): passphrase is currently not used since there's no keybase yet, the prompt is here to mimick the real world UX
+			pwd = readPassphrase(pwd)
+
+			msg := &typesUtil.MessageUnpause{
+				Address:   pk.Address(),
+				Signer:    pk.Address(),
+				ActorType: cmdDef.ActorType,
+			}
+
+			tx, err := prepareTxJson(msg, pk)
+			if err != nil {
+				return err
+			}
+
+			resp, err := postRawTx(cmd.Context(), pk, tx)
+			if err != nil {
+				return err
+			}
+			// DISCUSS(#310): define UX for return values - should we return the raw response or a parsed/human readable response? For now, I am simply printing to stdout
+			fmt.Println(resp)
+
+			return nil
+		},
+	}
+	return unpauseCmd
+}
+
+func readPassphrase(currPwd string) string {
```

Comment:
Can move functions below to util file?

Reply:
https://github.com/pokt-network/pocket/pull/177/commits/c7406148de537865cd84dd0188a6568f81a9bbb8